### ISSUE TITLE
Passthrough effect drift from mesh

### ISSUE THREAD

**Issue by HeberRyan** [oculus-samples/Unreal-PassthroughSample#5]
![Passthrough Image drift from mesh](https://github.com/oculus-samples/Unreal-PassthroughSample/assets/114383810/b1eb4d0d-336c-4ef8-8390-8de0a2ebf4bc)
Has anyone encountered this kind of situation? Need help, Any help will be appreciated!

Thanks guys!

**Comment by wangpingsx** [comment]
Hi HeberRyan, Thank you for utilizing the passthrough feature. It's unfortunate to hear that the passthrough window isn't functioning as expected.

Regrettably, without access to your project, it's challenging to provide specific suggestions for a resolution. However, I recommend trying our sample project as an alternative approach:

1. If you're utilizing Epic Unreal Engine, please ensure you're using version 5.3 as it is the recommended version.
2. Make sure you have downloaded and installed the latest Meta XR Plugin from the following link: 
 https://developer.oculus.com/downloads/package/unreal-engine-5-integration
 3. Try this map:  https://github.com/oculus-samples/Unreal-PassthroughSample/blob/main/Content/Passthrough/Maps/PTSurfaceProjected.umap
 4. Additionally, we offer a straightforward step-by-step guide which I highly recommend trying out: https://developer.oculus.com/documentation/unreal/unreal-passthrough-window-tutorial/ 
 
Feel free to explore these resources, and hopefully, they will assist you in resolving the issue with the passthrough feature.

Best regards,
Peter

**Comment by HeberRyan** [comment]
Hi Peter

Thanks for your reply!

I found out the problem was due to my pawn. I've reset Orientation and Position in player pawn in BeginPlay and forgot it.

BTW, I also wanna know why the plane mesh from the Engine doesn't work in the passthrough window but the plane from the sample worked (by adding the static mesh component to Oculus XRPassthrough Layer). what's the difference between the engine plane and the passthrough sample plane?

Best
Ryan
